### PR TITLE
Prepare for Hubble CLI v0.9.0 release

### PR DESCRIPTION
**Pull request by gandro**
See commits and https://github.com/cilium/hubble/blob/master/RELEASE.md regarding the release process.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,93 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v0.9.0] - 2021-11-30
+
+Hubble v0.9.0 coincides with Cilium v1.11. It brings many improvements to the
+CLI: Colored output (#551), improved readability and alternative output formats
+in `hubble status` (#629, #614), and the ability specify custom filters via the
```
Comment (rolinh) [inline CHANGELOG.md:13]:
```suggestion
in `hubble status` (#629, #614), and the ability to specify custom filters via the
```

**Review by rolinh — APPROVED** [review]
:100:  (except a missing word in the notes)

File: CHANGELOG.md (hunk 1 above)
Comment (gandro) [reply CHANGELOG.md]: Thanks! Fixed!

**Comment by glibsm** [comment]
Since the release technically goes `1.16.3 -> 1.17.3` maybe we should consider some sort of basic mechanism for squashing these.

```
* Update Go to 1.16.4 (#548, @tklauser)
* Update Go to 1.16.5 (#564, @tklauser)
* Update Go to 1.16.7 (#604, @tklauser)
* Update Go to 1.17 (#612, @tklauser)
* Update Go to 1.17.2 (#635, @tklauser)
* update Go to v1.16.6, alpine to 3.14 (#585, @rolinh)
* update Go to v1.17.1 (#620, @rolinh)
* Update Go to v1.17.3, golangci-lint to v1.43.0 (#646, @rolinh)
```

Similar story with some other things
```
* build(deps): bump google.golang.org/grpc from 1.38.0 to 1.39.0 (#584, @dependabot[bot])
* build(deps): bump google.golang.org/grpc from 1.39.0 to 1.39.1 (#608, @dependabot[bot])
* build(deps): bump google.golang.org/grpc from 1.39.1 to 1.40.0 (#610, @dependabot[bot])
* build(deps): bump google.golang.org/grpc from 1.40.0 to 1.41.0 (#634, @dependabot[bot])
* build(deps): bump google.golang.org/grpc from 1.41.0 to 1.42.0 (#649, @dependabot[bot])
```

Manual editing is an option, but it would be great if we can avoid it somehow

**Comment by gandro** [comment]
Agreed. Though there is also the case to be made for individual PRs, as some of them will sometimes contain custom notes as well or update multiple things at once (such as e.g. ##585)

**Pull request merged**